lua/plugins/filetype.lua

```diff
@@ -46,7 +46,7 @@ vis.ftdetect.filetypes = {
 	},
 	bash = {
 		utility = { "^[db]ash$", "^sh$","^t?csh$","^zsh$" },
-		ext = { "%.bash$", "%.csh$", "%.sh$", "%.zsh$" ,"^APKBUILD$", "%.ebuild$", "^.bashrc$", "^.bash_profile$" },
+		ext = { "%.bash$", "%.csh$", "%.sh$", "%.zsh$" ,"^APKBUILD$", "%.ebuild$", "^.*shrc$", "^.sh_profile$" },
 		mime = { "text/x-shellscript", "application/x-shellscript" },
```

Comment:
And what about the good old `~/.profile`? Also, aren’t you missing `*` in last item? Shouldn’t it be `^.*sh_profile$`?

Reply:
I think instead of using a wildcard, the extensions would be explicitly matched. I'll update them.